ui/pages/cluster_config_page/cluster_config_page.go

```diff
@@ -0,0 +1,215 @@
+package cluster_config_page
+
+import (
+	"fmt"
+	"ktea/config"
+	"ktea/kadmin"
+	"ktea/kontext"
+	"ktea/ui"
+	"ktea/ui/components/statusbar"
+	ktable "ktea/ui/components/table"
+
+	"github.com/charmbracelet/bubbles/spinner"
+	"github.com/charmbracelet/bubbles/table"
+	"github.com/charmbracelet/bubbles/viewport"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/charmbracelet/log"
+	"gopkg.in/yaml.v3"
+)
+
+type state int
+
+const (
+	loadingState state = iota
+	loadedState
+	errorState
+)
+
+type viewMode int
+
+const (
+	brokerListMode viewMode = iota
+	brokerDetailMode
+)
+
+type Model struct {
+	state    state
+	spinner  spinner.Model
+	viewport *viewport.Model
+	cluster  *config.Cluster
+	configs  kadmin.ClusterConfig
+	err      error
+
+	mode           viewMode
+	brokerTable    table.Model
+	selectedBroker *kadmin.BrokerConfig
+	ka             kadmin.Kadmin
+}
+
+func New(cluster *config.Cluster, ka kadmin.Kadmin) (*Model, tea.Cmd) {
+	s := spinner.New()
+	s.Spinner = spinner.Dot
+
+	t := ktable.NewDefaultTable()
+	t.SetColumns([]table.Column{
+		{Title: "ID", Width: 5},
+		{Title: "Address", Width: 30},
+	})
+
+	return &Model{
+		state:       loadingState,
+		spinner:     s,
+		cluster:     cluster,
+		mode:        brokerListMode,
+		brokerTable: t,
+		ka:          ka,
+	}, func() tea.Msg {
```

Comment:
When the kadmin methods return a `tea.Msg` you could just simply `return model, kadmin.GetClusterConfig`